blog/service/impl/src/main/kotlin/com/excref/kotblog/blog/service/category/impl/CategoryServiceImpl.kt

```diff
@@ -0,0 +1,75 @@
+package com.excref.kotblog.blog.service.category.impl
+
+import com.excref.kotblog.blog.persistence.category.CategoryRepository
+import com.excref.kotblog.blog.service.category.CategoryService
+import com.excref.kotblog.blog.service.category.domain.Category
+import com.excref.kotblog.blog.service.category.exception.CategoryAlreadyExistsForNameException
+import com.excref.kotblog.blog.service.category.exception.CategoryNotExistsForUuidException
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+/**
+ * @author Ruben Vardanyan
+ * @since 06/07/2017 12:31
+ */
+@Service
+class CategoryServiceImpl : CategoryService {
+
+    init {
```

Comment:
Let's discuss do we really need this garbage code? 😃  I think we can remove it from the project?